Prebuilt toolchains error out with docker sandbox

The new 0.1.0 downloaded tools work with default sandbox, but error out for me with docker sandbox, producing this error:
```
CMake Error: Could not find CMAKE_ROOT !!!
CMake has most likely not been installed correctly.
Modules directory not found in
/execroot/myproject/external/cmake-3.19.6-Linux-x86_64/share/cmake-3.19
CMake Error: Error executing cmake::LoadCache(). Aborting.
```
Using rbe-ubuntu-18-06 with bazel_toolchains 4.0.0

--- a/toolchains/prebuilt_toolchains.bzl
+++ b/toolchains/prebuilt_toolchains.bzl
@@ -28,16 +28,10 @@ filegroup(
     ),
 )
 
-filegroup(
-    name = "cmake_bin",
-    srcs = ["bin/{bin}"],
-    data = [":cmake_data"],
-)
-
 native_tool_toolchain(
     name = "cmake_tool",
-    path = "$(execpath :cmake_bin)",
-    target = ":cmake_bin",
+    path = "bin/{bin}",
+    target = ":cmake_data",
 )
 """
 

--- a/toolchains/prebuilt_toolchains.py
+++ b/toolchains/prebuilt_toolchains.py
@@ -133,16 +133,10 @@ filegroup(
     ),
 )
 
-filegroup(
-    name = "cmake_bin",
-    srcs = ["bin/{{bin}}"],
-    data = [":cmake_data"],
-)
-
 native_tool_toolchain(
     name = "cmake_tool",
-    path = "$(execpath :cmake_bin)",
-    target = ":cmake_bin",
+    path = "bin/{bin}",
+    target = ":cmake_data",
 )
 \"\"\"
 

--- a/toolchains/prebuilt_toolchains_repository.bzl
+++ b/toolchains/prebuilt_toolchains_repository.bzl
@@ -7,16 +7,10 @@ load("@rules_foreign_cc//tools/build_defs/native_tools:native_tools_toolchain.bz
 """
 
 _TOOLCHAIN = """\
-native_tool_toolchain(
-    name = "{repo}_tool",
-    path = "$(execpath @{repo}//:{tool}_bin)",
-    target = "@{repo}//:{tool}_bin",
-)
-
 toolchain(
     name = "{repo}_toolchain",
     exec_compatible_with = {exec_compatible_with},
-    toolchain = ":{repo}_tool",
+    toolchain = "@{repo}//:{tool}_tool",
     toolchain_type = "@rules_foreign_cc//tools/build_defs:{tool}_toolchain",
 )
 """

